hack/tas-kind-cluster.yaml

```diff
@@ -0,0 +1,155 @@
+---
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+    kubeadmConfigPatches:
+      - |
+        kind: ClusterConfiguration
+        apiVersion: kubeadm.k8s.io/v1beta3
+        scheduler:
+          extraArgs:
+            v: "2"
+        controllerManager:
+          extraArgs:
+            v: "2"
+        apiServer:
+          extraArgs:
+            enable-aggregator-routing: "true"
+            v: "2"
+  - role: worker
+    labels:
+      instance-type: on-demand
```

Comment:
I don't think this is useful in this case

Reply:
Oh I see, you use it for the RF configuration: https://github.com/kubernetes-sigs/kueue/pull/3489/files/85b53b86d0a4e59f15c23044cff82a4ddf35c885#diff-3023a038cb1a4d2e28eea7bc62407bd3a86e900e5aeccb3f3778ca6d553034dbR74

Use something more realistically used, like "cloud.provider.com/node-group: tas-group"